src/nrf_802154.h

```diff
@@ -1002,7 +1002,30 @@ void nrf_802154_pan_coord_set(bool enabled);
 bool nrf_802154_pan_coord_get(void);
 
 /**
- * @brief Adds the address of a peer node for which the provided ACK data is to be set.
+ * @brief Select the source matching algorithm.
+ *
+ * @note This method should be called after driver initialization, but before transceiver is enabled.
+ *
+ * When calling @ref nrf_802154_ack_data_pending_bit_should_be_set method, it will choose one of
+ * several source address matching algorithms. In order for specific algorithm to be chosen, this
+ * function should be called beforehand.
+ *
+ * @param[in]  match_method Source address matching method to be used.
+ */
+void nrf_802154_src_addr_matching_method_set(nrf_802154_src_addr_match_t match_method);
+
+/**
+ * @brief Adds the address of a peer node for which the provided ACK data
+ * is to be added to the pending bit list.
+ *
+ * Pending bit list works differently, depending on upper layer for which the source
+ * address matching method is selected.
+ * In case of Thread, pending bit is set only for the addresses found in the list.
+ * In case of Zigbee, pending bit is cleared only for the short addresses found in the list.
+ * In case of standard-compliant method, pending bit is always true, which requires
+ * empty data frame with AR set to 0 to be transmitted immediately afterwards.
+ *
+ * Used method can be set during initialization phase by calling @ref nrf_802154_src_matching_method.

```

Comment:
```suggestion
 * The source address matching method can be set during the initialization phase by calling @ref nrf_802154_src_matching_method.
```
`used method`? meaning `method to be used`?